src/channel/coolholepoints.js

```diff
@@ -458,11 +519,12 @@ class Coolpoints extends ChannelModule {
           `Wise spender ${user.getName()} spent ${pointsToSpend} points on ${action}`,
           new ReturnPointData(
             user.getName(),
-            pointsToSpend,
+            -pointsToSpend,
```

Comment:
found by @esnyder232 , UI was showing "green' when we were sending positive values (duh)